lib/util/meta.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var META_SYMBOL = Symbol('json-schema-ref-parser meta');
+
+exports.META_SYMBOL = META_SYMBOL;
+
+/**
+ * Safely sets meta property on the given object value
+ *
+ * @param  {object} obj - A object to set meta info on
+ * @param  {string} prop - Name of property
+ * @param  {string} prop - Value to set
+ */
+exports.setMeta = function (obj, prop, value) {
```

Comment:
Let's rename `setMeta()` to `setMetadata()`. It's more explicit/clear.

```suggestion
exports.setMetadata = function (obj, prop, value) {
```